### PR TITLE
Update file-system.md (PR#1584 fixes)

### PR DESCRIPTION
**Pull request by omar-srbd**
### Change Description ###

This PR was created to implement the changes that was suggested in #1584 but could not be implemented by the author.

Signed off by: omar.saeed@samsung.com

### CODE REVIEW

File: docs/application/web/guides/data/file-system.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # File System
 
-> **Note**  
+> [!NOTE]  
 > 
 > With the introduction of Tizen version 5.0, the Filesystem API has undergone a major overhaul. Many of the existing methods are deprecated and new methods are introduced. For information on the previous versions of API, see [Deprecated Filesystem Guide](./file-system-old.md). This guide explains about the APIs for Tizen 5.0 and its newly introduced methods.
```
Comment (mijin85cho) [inline docs/application/web/guides/data/file-system.md:5]:
```suggestion
> [!NOTE]  
> With the introduction of Tizen version 5.0, the Filesystem API has undergone a major overhaul. Many of the existing methods are deprecated and new methods are introduced. For information on the previous versions of API, see [Deprecated Filesystem Guide](./file-system-old.md). This guide explains about the APIs for Tizen 5.0 and its newly introduced methods.
```

Global.

It is confusable, but the new style note delete the middle empty line.

https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning

Comment (omar-srbd) [reply docs/application/web/guides/data/file-system.md]: @mijin85cho, fixed all the note related issues.

File: docs/application/web/guides/data/file-system.md [hunk 2]
```diff
@@ -238,16 +236,15 @@ You can retrieve a list of files or file URIs using the `listDirectory()` and `t
...
+   - Close the file when it is not in use:
        ```
        fileHandleRead.close();
        ```
 
-> **Note**
->
-> When you use readString(), readData(), readBlob() or one of their NonBlocking equivalent, FileHandle position is set right where the reading operation ended. To read file again from the beggining, use seek() method on FileHandle object.
+        > [!NOTE]
+        > When you use readString(), readData(), readBlob() or one of their NonBlocking equivalent, FileHandle position is set right where the reading operation ended. To read file again from the beggining, use seek() method on FileHandle object.
```
Comment (safir-srbd) [inline docs/application/web/guides/data/file-system.md:245]:
```suggestion
        > When you use readString(), readData(), readBlob(), or one of their NonBlocking equivalents, FileHandle position is set right where the reading operation ended. To read the file again from the beginning, use seek() method on FileHandle object.
```

File: docs/application/web/guides/data/file-system.md [hunk 3]
```diff
@@ -259,11 +256,10 @@ To write to files:
 
    The method returns a `FileHandle` object, which is a handle to the opened or newly created file.
 
-   > **Note**
-   >
+   > [!NOTE]
    > openFile() method used with mode 'w' creates new file or opens existing file and erases its contents. Any data contained in a file opened in that mode will be lost.
```
Comment (safir-srbd) [inline docs/application/web/guides/data/file-system.md:260]:
```suggestion
   > openFile() method used with mode 'w' creates a new file or opens an existing file and erases its contents. Any data contained in a file opened in that mode will be lost.
```

**Review by safir-srbd — COMMENTED** [review]
@omar-srbd I have provided suggestions. For those lines, for which I could not directly suggest, here are the changes:

File: docs/application/web/guides/data/file-system.md

Line 251 - .....object based on a curent position, which can be obtained or modified through calling `seek()` operation on `FileHandle` object.
Suggestion - .....object based on a **current** position, which can be obtained or modified through calling `seek()` operation on `FileHandle` object.

Line 337 - Since the third parameter is set to `true`, existing file or directory with the same name in the target directory is overwritten:
Suggestion - Since the third parameter is set to `true`, **an** existing file or directory with the same name in the target directory is overwritten: 

Line 351 - The following example changes file name to the `newName.txt`:
Suggestion - The following example changes **the** file name to `newName.txt`:

**Review by safir-srbd — APPROVED** [review]
@omar-srbd Thank you for making the changes I suggested. Everything looks good on my end.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Pull request merged**